docs/dev/table/tableApi.md

```diff
@@ -2993,6 +2993,30 @@ MAP.at(ANY)
   </tbody>
 </table>
 
+<table class="table table-bordered">
+  <thead>
+    <tr>
+      <th class="text-left" style="width: 40%">Row functions</th>
+      <th class="text-center">Description</th>
+    </tr>
+  </thead>
+
+  <tbody>
+
+    <tr>
+      <td>
+        {% highlight java %}
+row(ANY, [, ANY]*)
+{% endhighlight %}
+      </td>
+      <td>
+        <p>Creates a row from a list of values. row can be access via <a href="tableApi.html#built-in-functions">built-in functions</a> (see Value access functions section) </p>
```

Comment:
`A row is a composite type that can row can be access via <a href="tableApi.html#built-in-functions">value access functions</a>.`